nodewatcher/core/generator/cgm/devices.py

```diff
@@ -529,6 +529,11 @@ class DeviceBase(object):
     radios = None
     switches = None
     usb = False
+    usb = None
```

Comment:
You don't need multiple of these, this is just the default value and it can be either `None` or `False`. Keep just one (e.g., `usb = None`) and remove the rest.